sklearn/metrics/cluster/supervised.py

```diff
@@ -36,14 +36,18 @@ def check_clusterings(labels_true, labels_pred):
 
     Parameters
     ----------
-    labels_true : int array, shape = [n_samples]
-        The true labels
+    labels_true : array-like of shape (n_samples,)
+        The true labels.
 
-    labels_pred : int array, shape = [n_samples]
-        The predicted labels
+    labels_pred : array-like of shape (n_samples,)
+        The predicted labels.
     """
-    labels_true = np.asarray(labels_true)
-    labels_pred = np.asarray(labels_pred)
+    labels_true = check_array(
+        labels_true, ensure_2d=False, ensure_min_samples=0
```

Comment:
@amueller note that the test can pass if I allow for 0 sample. I think it was more a general question (but this is a long time ago).